draft-ietf-quic-recovery.md

```diff
@@ -1541,9 +1547,10 @@ newly acked_packets from sent_packets.
       congestion_window += acked_packet.sent_bytes
     else:
       // Congestion avoidance.
-      congestion_window +=
-        max_datagram_size * acked_packet.sent_bytes
-        / congestion_window
+      bytes_acked += acked_packet.sent_bytes
+      if (bytes_acked >= congestion_window):
+        bytes_acked -= congestion_window
+        congestion_window += max_datagram_size
```

Comment:
There's a minor behavior difference here from the old code, in that the cwnd here increases by at most 1. I think that's fine; I don't think it's possible to have an increase that is larger. The following code is the same as the old behavior, but I don't think it's necessary.

```suggestion
      while (bytes_acked >= congestion_window):
        bytes_acked -= congestion_window
        congestion_window += max_datagram_size
```

Reply:
I don't think `while()` is necessary because `OnPacketAcked()` (recently changed) is called per acked packet, so `acked_packet.sent_bytes` is always smaller than `max_datagram_size`.

Reply:
I agree with @junhochoi.

Reply:
Agreed. I wanted to note the difference, but as I said, I don't think it's necessary.